assets/javascripts/discourse/templates/components/chat-channel-preview-card.hbs

```diff
@@ -0,0 +1,16 @@
+<div class={{cardClasses}}>
+  {{chat-channel-title channel=channel}}
+  {{#if hasDescription}}
+    <p class="chat-channel-preview-card__description">{{channel.description}}</p>
+  {{/if}}
+  {{d-button
+    action=(action "onJoinChannel")
+    label="chat.channel_settings.join_channel"
+    class="chat-channel-preview-card__join-channel-btn btn-primary"
+  }}
+  <p class="chat-channel-preview-card__browse-all">
```

Comment:
Why is this wrapped in a p-tag? It's semantically not correct and it adds margin we don't want.
I would prefer the a-tag directly, without a wrapping element, but if for some ember-reason you need one, I'd prefer a simple span.

Reply:
No reason particularly, will change :)

Reply:
There is some margin/padding in the design that is lost if I remove the P tag though:

![image](https://user-images.githubusercontent.com/920448/178207170-6d1ad420-f7d0-459a-99a8-af4840a7beb0.png)

Here is what it looks like locally now:

![image](https://user-images.githubusercontent.com/920448/178207209-96fa3af1-cd2f-4ad1-829c-932016ab0dbf.png)

If you want you can do another round of tweaks and push to this PR? I will just remove the P tag